### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by h0lyalg0rithm**
Use high res badges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ fog is the Ruby cloud services library, top to bottom:
...
-[![Build Status](https://secure.travis-ci.org/fog/fog.png?branch=master)](http://travis-ci.org/fog/fog)
-[![Dependency Status](https://gemnasium.com/fog/fog.png)](https://gemnasium.com/fog/fog)
-[![Code Climate](https://codeclimate.com/github/fog/fog.png)](https://codeclimate.com/github/fog/fog)
-[![Gem Version](https://fury-badge.herokuapp.com/rb/fog.png)](http://badge.fury.io/rb/fog)
-[![Gittip](http://img.shields.io/gittip/geemus.png)](https://www.gittip.com/geemus/)
+[![Build Status](https://secure.travis-ci.org/fog/fog.svg?branch=master)](http://travis-ci.org/fog/fog)
+[![Dependency Status](https://gemnasium.com/fog/fog.svg)](https://gemnasium.com/fog/fog)
+[![Code Climate](https://codeclimate.com/github/fog/fog/badges/gpa.svg)](https://codeclimate.com/github/fog/fog)
+[![Gem Version](https://badge.fury.io/rb/fog.svg)](http://badge.fury.io/rb/fog)
+[![Gittip](https://img.shields.io/gratipay/user/dougwilson.svg)](https://www.gittip.com/geemus/)
```
Comment (geemus) [inline README.md]: This points to a different user now. Really I think we could just drop that altogether though, I don't think anyone has really ever used it.

**Comment by geemus** [comment]
One small comment, otherwise looking good, thanks!

**Comment by h0lyalg0rithm** [comment]
Got rid of gittip

**Comment by geemus** [comment]
Thanks!

**Pull request merged**